plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestingIgniteServer.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.ignite;
+
+import java.io.Closeable;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+
+import static io.trino.plugin.ignite.TestIgniteContainer.HTTP_PORT;
+import static java.lang.String.format;
+
+public class TestingIgniteServer
+        implements Closeable
+{
+    private static final String IGNITE_IMAGE = "apacheignite/ignite:2.10.0";
```

Comment:
This seems to be the latest release version (excluding the alpha). Can we test using a lower version by default since most users won't be running the latest version yet (it was released in March 2021 which is pretty recent).

Reply:
Currently, changed the version to 2.8.0. not sure it is lower enough.